package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/CameraImporter.cs

```diff
@@ -25,13 +26,18 @@ public static class CameraImporter {
     /// <summary>
     /// Copy camera data from USD to Unity with the given import options.
     /// </summary>
-    public static void BuildCamera(CameraSample usdCamera,
-                                 GameObject go,
-                                 SceneImportOptions options) {
-      var cam = ImporterBase.GetOrAddComponent<Camera>(go);
-      usdCamera.CopyToCamera(cam, setTransform: false);
-      cam.nearClipPlane *= options.scale;
-      cam.farClipPlane *= options.scale;
+    public static void BuildCamera(pxr.SdfPath path,
+      CameraSample usdCamera,
+      GameObject go,
+      SceneImportOptions options, Scene scene)
+    {
+      if (scene.AccessMask == null || scene.IsPopulatingAccessMask || scene.AccessMask.Included.ContainsKey(path))
+      {
+        var cam = ImporterBase.GetOrAddComponent<Camera>(go);
+        usdCamera.CopyToCamera(cam, false);
```

Comment:
This will update all camera attributes whether they are animated or not. I think we should only update the attributes that are indeed dynamic (the ones in the accessMask).
What do you think?

Reply:
I think the all-or-nothing implementation is a good place to start. I would definitely profile to see the impact of making it more granular -- I'm not opposed to it, but it's good practice any time you make a change for performance.

Reply:
After discussing with Vlad, given we are creating a GfCamera from all cam attributes anyway, there's no direct way to go component by component beside duplicating the GfCamera code. So let's stick to the all-or-nothing approach